CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
-# delayed_job_chainable_hooks
+# Changelog
 
-## v1.0.0
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## Unreleased
+
+## 2.0.0 - 2021-03-31
+### Changed
+- Require Ruby 2.7+
```

Comment:
Most of our libraries require Ruby 2.6+ since Ruby 2.6 won't be EOL until March 31, 2022. How painful is support Ruby 2.6?

Reply:
It should be straightforward to include.